### ISSUE TITLE
Problematic ads

### ISSUE THREAD

**Issue by Alex-302** [AdguardTeam/AdguardFilters#7093]
The list of popular ads, which can not be blocked by browser extensions:
- [instartlogic script](https://github.com/AdguardTeam/AdguardFilters/issues/3844)
- Yandex direct. For example: otzovik.com, e1.ru, dni.ru,kakprosto.ru, ***Yandex Pogoda***

**Comment by ameshkov** [comment]
Partly we'll be able to solve it with $replace and $$ rules support in the FF add-on. We should figure a solution for other browsers as well.

**Comment by AdamWr** [comment]
Some type of video ads can't be blocked (usually these which use VAST), because it will trigger anti-adblock.
For example - https://github.com/AdguardTeam/AdguardFilters/issues/7042

**Comment by seba2282** [comment]
I have not typical problem after update Windows 10 to Fall Creators Update. I tried Adguard and Ublock Origin on www.miniminiplus.pl/filmy and www.teletoonplus.pl/filmy and the both have the same problem as extensions for MS Edge. After watching few videos I will see ads video before normal video in player. This ads have 30 s and I know, they are from redcdn.pl subdomain.

**Comment by Alex-302** [comment]
#7093 is not for specific ads. Open separate issue.
